Framework/src/InfrastructureGenerator.cxx

```diff
@@ -128,7 +128,8 @@ void InfrastructureGenerator::customizeInfrastructure(std::vector<framework::Com
 void InfrastructureGenerator::printVersion()
 {
   // Log the version
-  QcInfoLogger::GetInstance() << "QC version " << o2::quality_control::core::Version::getString() << infologger::endm;;
+  QcInfoLogger::GetInstance() << "QC version " << o2::quality_control::core::Version::getString() << infologger::endm;
+  ;
```

Comment:
why semicolon?

Reply:
thanks for noticing it. It is just a typo.